Xamarin.Forms.Platform.WinRT/LabelRenderer.cs

```diff
@@ -72,15 +72,31 @@ public override SizeRequest GetDesiredSize(double widthConstraint, double height
 				_perfectSizeValid = true;
 			}
 
-			if (widthConstraint >= _perfectSize.Request.Width && heightConstraint >= _perfectSize.Request.Height)
+			var widthFits = widthConstraint >= _perfectSize.Request.Width;
```

Comment:
This bug did not affect Windows, but the logic here was copied from iOS, so I copied the fix over for housekeeping purposes.